azplugins/module.cc

```diff
@@ -226,10 +236,21 @@ PYBIND11_MODULE(_azplugins, m)
     azplugins::detail::export_PositionRestraintComputeGPU(m);
     #endif // ENABLE_CUDA
 
-    /* MPCD */
+
+    /* MPCD components */
     #ifdef ENABLE_MPCD
+    azplugins::detail::export_SinusoidalChannel(m);
+    azplugins::detail::export_SinusoidalExpansionConstriction(m);
+    azplugins::detail::export_SinusoidalChannelFiller(m);
+    azplugins::detail::export_SinusoidalExpansionConstrictionFiller(m);
+    mpcd::detail::export_ConfinedStreamingMethod<azplugins::detail::SinusoidalChannel>(m);
+    mpcd::detail::export_ConfinedStreamingMethod<azplugins::detail::SinusoidalExpansionConstriction>(m);
     azplugins::detail::export_MPCDReversePerturbationFlow(m);
     #ifdef ENABLE_CUDA
+    azplugins::detail::export_SinusoidalChannelFillerGPU(m);
+    azplugins::detail::export_SinusoidalExpansionConstrictionFillerGPU(m);
+    mpcd::detail::export_ConfinedStreamingMethodGPU<azplugins::detail::SinusoidalChannel>(m);
+    mpcd::detail::export_ConfinedStreamingMethodGPU<azplugins::detail::SinusoidalExpansionConstriction>(m);
```

Comment:
I think we probably should comment these out so that they do not compile at all (if they do not work)